easybuild/tools/options.py

```diff
@@ -177,6 +178,7 @@ def override_options(self):
             'allow-modules-tool-mismatch': ("Allow mismatch of modules tool and definition of 'module' function",
                                             None, 'store_true', False),
             'cleanup-builddir': ("Cleanup build dir after successful installation.", None, 'store_true', True),
+            'compile-lmod-caches': ("Compile Lmod cache files with luac after updating them", None, 'store_true', False),
```

Comment:
what does this do? should be just a toggle, that if you run `--update-cache` also compiles it (but i don't like the idea. lmod should do what is best, i.e. compile if luac is avail).